client/components/searchResult.txt

```diff
@@ -0,0 +1,891 @@
+{
+  id: '4Vmq51yYzlDEzCQ03zPsGg',
```

Comment:
please put this file in a folder named `fixtures/` and change it from a `.txt` to a `.json` file.